app/src/main/java/woowacourse/movie/presentation/view/main/home/seatpick/model/SeatGradeModel.kt

```diff
@@ -1,8 +1,10 @@
 package woowacourse.movie.presentation.view.main.home.seatpick.model
 
-enum class SeatGradeModel(val seatColor: String) {
-    S_CLASS("#19D358"),
-    A_CLASS("#1B48E9"),
-    B_CLASS("#E9B21B"),
-    NONE("#FFFFFF");
+import woowacourse.movie.R
+
+enum class SeatGradeModel(val seatColor: Int) {
```

Comment:
```suggestion
enum class SeatGradeModel(@ColorRes val seatColor: Int) {
```
color의 리소스임을 명시해도 좋아요